src/main/java-templates/org/postgresql/ds/PGConnectionPoolDataSource.java

```diff
@@ -29,7 +29,7 @@
  * @author Aaron Mulder (ammulder@chariotsolutions.com)
  */
 public class PGConnectionPoolDataSource
-    extends @CONN_POOL_DS_CLASS@
+    extends ${CONN_POOL_DS_CLASS}
```

Comment:
@vlsi Looking further into the template methods in Driver.java, I think JDBC4 is already the minimum. Therefore the change could be made to simplify that part. Maybe it would be better to make these changes separate PRs once this is merged? Therefore this change can purely move the build from Ant to Maven as the change of source folders has already generated many changed files.